src/main/java/com/hrznstudio/galacticraft/misc/capes/CapesLoader.java

```diff
@@ -24,53 +24,36 @@
 
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
-import com.hrznstudio.galacticraft.api.capes.CapeListener;
-import com.hrznstudio.galacticraft.api.capes.models.CapesModel;
+import com.hrznstudio.galacticraft.Galacticraft;
+import net.minecraft.util.Util;
 import org.apache.commons.io.IOUtils;
 
 import java.io.IOException;
 import java.net.URL;
 import java.nio.charset.Charset;
-import java.util.ArrayList;
-import java.util.List;
+import java.util.Map;
 
-/**
- * @author <a href="https://github.com/StellarHorizons">StellarHorizons</a>
- */
-public class CapeLoader {
-
-    private final List<CapeListener> listeners;
-    private final Gson gson = new GsonBuilder().create();
-
-    public CapeLoader() {
-        this.listeners = new ArrayList<>();
-    }
+public class CapesLoader {
+    public static Map PLAYERS;
 
-    public boolean register(CapeListener listener) {
-        if (!this.listeners.contains(listener)) {
-            this.listeners.add(listener);
-            return true;
-        } else {
-            return false;
-        }
-    }
+    public static void load() {
+        Util.getMainWorkerExecutor().execute(() -> {
+            long startLoad = System.currentTimeMillis();
+            Gson gson = new GsonBuilder().create();
+            Galacticraft.logger.info("[Galacticraft] Loading capes data...");
```

Comment:
There is a `LOGGER_PREFIX` at the bottom of the `Constant` class now

Reply:
resolved in 90722cb